sender.py

```diff
@@ -0,0 +1,24 @@
+from generate_temperature import generate_temperature_samples
+from generate_charge_rate import generate_charge_rate_samples
+
+
+def main():
+    # Totally 10 samples, each sample consisting of 60 readings for each parameters
+    NUM_OF_READINGS = 60
+    NUM_OF_SAMPLES = 10
+
+    # Outputting the Charge rates reading to standard output console
+    for _ in range(NUM_OF_SAMPLES):
+        charge_rates = generate_charge_rate_samples(NUM_OF_READINGS)
+
+        print(",".join([str(i) for i in charge_rates]))
+
+    # Outputting the Temperature reading to standard output console
+    for _ in range(NUM_OF_SAMPLES):
+        temperatures = generate_temperature_samples(NUM_OF_READINGS)
+
+        print(",".join([str(i) for i in temperatures]))
```

Comment:
Data can be sent line-by-line to the console as per the requirement

Reply:
I am sending 1 stream of 60 readings in a line. Such 10 streams are being sent for 2 battery parameters seperately.